api/tests/PoolCandidateTest.php

```diff
@@ -0,0 +1,606 @@
+<?php
+
+use App\Models\Classification;
+use App\Models\CmoAsset;
+use App\Models\OperationalRequirement;
+use App\Models\Pool;
+use App\Models\PoolCandidate;
+use GraphQL\Type\Definition\EnumType;
+use Laravel\Lumen\Testing\DatabaseMigrations;
+use Nuwave\Lighthouse\Testing\ClearsSchemaCache;
+use Nuwave\Lighthouse\Testing\MakesGraphQLRequestsLumen;
+// use PHPUnit\Framework\TestCase;
+
+class PoolCandidateTest extends TestCase
+{
+  use DatabaseMigrations;
+  use MakesGraphQLRequestsLumen;
+  use ClearsSchemaCache;
+
+  protected function setUp(): void
+  {
+    parent::setUp();
+    $this->bootClearsSchemaCache();
+  }
+
+  public function testFilterByClassification(): void
+  {
+
+    // Create 5 pool candidates.
+    Classification::factory()->count(3)->create();
+    PoolCandidate::factory()->count(5)->create();
+
+    // Create new classification and attach to 2 new pool candidates.
+    $classification = Classification::factory()->create([
+      'group' => 'CS',
```

Comment:
It might be better to create control classifications for testing that are less likely to be generated by the faker factory, like ZZ or something.